modules/core/src/gpu-type-utils/vertex-format-from-attribute.ts

```diff
@@ -77,6 +77,14 @@ export function getVertexFormatFromAttribute(
   const components = size as 1 | 2 | 3 | 4;
   let dataType: DataType | DataTypeNorm = getDataTypeFromTypedArray(typedArray);
 
+  // TODO - Special cases for WebGL (not supported on WebGPU), overrides the check below
+  if (dataType === 'uint8' && normalized && components === 1) {
```

Comment:
Do we plan to deprecate this at some point? What will happen if the app tries to run with WebGPU?

Reply:
> Do we plan to deprecate this at some point? What will happen if the app tries to run with WebGPU?

Yes. I had removed this in 9.1 for that exact reason, but it was breaking 3D tiles support in loaders.gl, so just re-enabling to unblock users until we have done necessary changes (converting these buffers to 2 or 4 components, either in GLTFLoader or luma)